.github/workflows/channels-ci.yml

```diff
@@ -31,12 +36,16 @@ jobs:
         # if: steps.npm-cache.outputs.cache-hit != 'true'
         run: |
           make node_modules
-          make e2e/playwright/node_modules
+          # make channels/e2e/playwright/node_modules
```

Comment:
I commented this out temporarily since we're no longer type checking the E2E tests in the web app right now. I've got a note to revisit that later